### PR TITLE
add Marshini Chetty at University of Chicago, remove retired, moved, non-affiliated Princeton faculty

### PR DESCRIPTION
**Pull request by feamster**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -13348,7 +13342,7 @@ Samir Ranjan Das,Stony Brook University,http://www3.cs.stonybrook.edu/~samir,NOS
 Samira Manabi Khan,University of Virginia,http://www.cs.virginia.edu/~smk9u,NOSCHOLARPAGE
 Samira Shaikh,UNC - Charlotte,https://analyticsfrontiers.uncc.edu/speakers/samira-shaikh,IM-64IoAAAAJ
 Samit Bhattacharya,IIT Guwahati,http://www.iitg.ernet.in/samit,oXu5o6gAAAAJ
-Samory Kpotufe,Princeton University,http://www.princeton.edu/~samory,NOSCHOLARPAGE
+Samory Kpotufe,Columbia University,http://www.columbia.edu/~skk2175,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings.csv:13345]: In Stats dept., not CS.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by feamster** [comment]
Done.

File: csrankings.csv (hunk 1 above)
Comment (feamster) [reply csrankings.csv:13345]: Fixed.

**Comment by emeryberger** [comment]
thanks!

**Comment by feamster** [comment]
Sure thing!

-Nick

**Pull request merged**